client/signup/steps/theme-selection/index.jsx

```diff
@@ -101,42 +122,27 @@ class ThemeSelectionStep extends Component {
 
 	render = () => {
 		const defaultDependencies = this.props.useHeadstart ? { themeSlugWithRepo: 'pub/twentysixteen' } : undefined;
+		const { translate } = this.props;
 
-		const pressableWrapperClassName = classNames( {
-			'theme-selection__pressable-wrapper': true,
-			'is-hidden': ! this.state.showPressable,
-		} );
-
-		const themesWrapperClassName = classNames( {
-			'theme-selection__themes-wrapper': true,
-			'is-hidden': this.state.showPressable,
-		} );
+		const headerText = this.state.showPressable
+			? translate( 'Upload your WordPress Theme' )
+			: translate( 'Choose a theme.' );
 
-		const { translate } = this.props;
+		const subHeaderText = this.state.showPressable
+			? translate( 'Our partner Pressable is here to help you' )
+			: translate( 'No need to overthink it. You can always switch to a different theme later.' );
```

Comment:
Maybe add context for translators. The other strings may not need it, but this one probably does.